src/Api/DTO/Request/ParcelShopSearchRequest.php

```diff
@@ -209,6 +210,6 @@ public function jsonSerialize()
             $request['id'] =  $this->parcelShopId;
         }
 
-        return $request;
+        return ArrayUtility::removeKeysWithEmptyValues($request);
```

Comment:
Could improve some of the code above as company and parcelShopId can be null now and they will be cleaned off, there's no need for if cases

Reply:
Good point